specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
           }
         }
       }
-    }
-  },
-  "definitions": {
-    "IconCode": {
-      "type": "integer",
-      "format": "int32",
-      "description": "Numeric value representing an image that displays the `iconPhrase`. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details.",
-      "enum": [
-        1,
-        2,
-        3,
-        4,
-        5,
-        6,
-        7,
-        8,
-        9,
-        10,
-        11,
-        12,
-        13,
-        14,
-        15,
-        16,
-        17,
-        18,
-        19,
-        20,
-        21,
-        22,
-        23,
-        24,
-        25,
-        26,
-        27,
-        28,
-        29,
-        30,
-        31,
-        32,
-        33,
-        34,
-        35,
-        36,
-        37,
-        38,
-        39,
-        40,
-        41,
-        42,
-        43,
-        44
-      ],
-      "x-ms-enum": {
-        "name": "IconCode",
-        "modelAsString": true,
-        "values": [
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Tropical Active Storms**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued active tropical storms. Information about the tropical storms includes, government ID, basin ID, year of origin, name and if it is subtropical.",
+        "operationId": "Weather_GetTropicalActiveStorms",
```

Comment:
This does not read naturally. `Weather_GetActiveTropicalStorms` makes more sense.

Reply:
@eriklindeman How do you think? This change will affect the title of document page.

Reply:
The names of the services should always, be 'Tropical Storm(s)', so I agree with that part, but I think it should be changed to 'Get Tropical Storms Active' to stay uniform with the other three Tropical Storm services if possible:

GetTropicalStormForecast
GetTropicalStormLocations
GetTropicalStormSearch

Reply:
Sure. I have updated the naming to GetTropicalStormsActive.